server-core/CMakeLists.txt

```diff
@@ -47,16 +47,9 @@ if(UNIX)
 endif(UNIX)
 
 find_package(asio CONFIG REQUIRED)
-target_link_libraries(server-cmd PRIVATE asio::asio)
-
 find_package(spdlog CONFIG REQUIRED)
-target_link_libraries(server-cmd PRIVATE spdlog::spdlog)
-
-find_package(Protobuf CONFIG REQUIRED)
-target_link_libraries(server-cmd PRIVATE protobuf::libprotobuf)
-
 find_package(cxxopts CONFIG REQUIRED)
-target_link_libraries(server-cmd PRIVATE cxxopts::cxxopts)
+target_link_libraries(server-cmd PRIVATE asio::asio spdlog::spdlog protobuf::libprotobuf cxxopts::cxxopts)

```

Comment:
`target_link_libraries` won't do link many times. It's a kind of declaration. After CMake configure the project, it will generate only one link command. One line or Multiline `target_link_libraries` have no difference.